packages/react/src/components/index.ts

```diff
@@ -1,40 +1,255 @@
-export * from './heading';
-export * from './input';
-export * from './label';
-export * from './link';
-export * from './list';
-export * from './pagination';
-export * from './paragraph';
-export * from './popover';
-export * from './radio';
-export * from './search';
-export * from './select';
-export * from './skeleton';
-export * from './skip-link';
-export * from './spinner';
-export * from './suggestion';
-export * from './switch';
-export * from './table';
-export * from './tabs';
-export * from './tag';
-export * from './textarea';
-export * from './textfield';
-export * from './toggle-group';
-export * from './tooltip';
-export * from './validation-message';
+export type { AlertProps } from './alert/alert';
+export { Alert } from './alert/alert';
+
+export type { AvatarProps } from './avatar/avatar';
+export { Avatar } from './avatar/avatar';
+
+export type { BadgePositionProps, BadgeProps } from './badge';
+export { Badge, BadgePosition } from './badge';
+
+export type {
+  BreadcrumbsItemProps,
+  BreadcrumbsLinkProps,
+  BreadcrumbsListProps,
+  BreadcrumbsProps,
+} from './breadcrumbs';
+export {
+  Breadcrumbs,
+  BreadcrumbsItem,
+  BreadcrumbsLink,
+  BreadcrumbsList,
+} from './breadcrumbs';
+
+export type { ButtonProps } from './button/button';
+export { Button } from './button/button';
+
+/* TODO: Remove this in the next major version */
+export type {
+  ComboboxEmptyProps,
+  ComboboxOptionProps,
+  ComboboxProps,
+} from './Combobox';
+export { Combobox, ComboboxEmpty, ComboboxOption } from './Combobox';
+
+export type { CardBlockProps, CardProps } from './card';
+export { Card, CardBlock } from './card';
+
+export type { CheckboxProps } from './checkbox/checkbox';
+export { Checkbox } from './checkbox/checkbox';
+
+export type {
+  ChipButtonProps,
+  ChipCheckboxProps,
+  ChipRadioProps,
+  ChipRemovableProps,
+} from './chip';
+export {
+  Chip,
+  ChipButton,
+  ChipCheckbox,
+  ChipRadio,
+  ChipRemovable,
+} from './chip';
+
+export type {
+  DetailsContentProps,
+  DetailsProps,
+  DetailsSummaryProps,
+} from './details';
+export { Details, DetailsContent, DetailsSummary } from './details';
+
+export type {
+  DialogBlockProps,
+  DialogProps,
+  DialogTriggerContextProps,
+  DialogTriggerProps,
+} from './dialog';
+export {
+  Dialog,
+  DialogBlock,
+  DialogTrigger,
+  DialogTriggerContext,
+} from './dialog';
+
+export type { DividerProps } from './divider/divider';
+export { Divider } from './divider/divider';
+
+export type {
+  DropdownButtonProps,
+  DropdownHeadingProps,
+  DropdownItemProps,
+  DropdownListProps,
+  DropdownProps,
+  DropdownTriggerContextProps,
+} from './dropdown';
+export {
+  Dropdown,
+  DropdownButton,
+  DropdownHeading,
+  DropdownItem,
+  DropdownList,
+  DropdownTrigger,
+  DropdownTriggerContext,
+} from './dropdown';
+
+export type { ErrorSummaryItemProps, ErrorSummaryProps } from './error-summary';
+export { ErrorSummary, ErrorSummaryItem } from './error-summary';
```

Comment:
Looks like we are missing some exports here? 

https://github.com/digdir/designsystemet/blob/7cbadc57ec5a74af6fe37b6d114570cfa45d9124/packages/react/src/components/error-summary/index.ts#L35